src/components/StudyList/StudyTable.js

```diff
@@ -123,6 +129,13 @@ const StudyTable = ({
             Files
           </Table.HeaderCell>
         );
+      case 'release':
+        return (
+          <Table.HeaderCell key={text}>
+            <Icon name="share square" color="grey" />
```

Comment:
Can we use the `tag` icon to stay consistent with iconography in the coordinator?